inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -100,17 +100,9 @@ std::vector<std::string> disabledTestPatterns() {
 
         // CPU plugin does not support some precisions
         R"(smoke_CachingSupportCase_CPU/LoadNetworkCacheTestBase.CompareWithRefImpl/ReadConcatSplitAssign_f32_batch1_CPU)",
-        // CPU plugin does not support some precisions
-        R"(.*Behavior.*OVExecGraphImportExportTest.*elementType=(i8|u32).*)",
-        R"(.*Behavior.*OVExecGraphImportExportTest.*elementType=(f16).*)",
-        R"(.*EltwiseLayerTest.*NetType=f16.*)",
-
-        // TODO: CVS-66526 overrides i/o precisions in execution graph
-        // as WA we used GetInputsInfo() precisions instead of ngraph ones
-        // R"(.*smoke_BehaviorTests.*OVExecGraphImportExportTest.*importExportedFunction.*type=(i16|u16).*)",
-        // R"(.*smoke_BehaviorTests.*OVExecGraphImportExportTest.*importExportedFunction.*type=(i64|u64).*)",
-        // R"(.*smoke_BehaviorTests.*OVExecGraphImportExportTest.*importExportedIENetwork.*type=(i16|u16).*)",
-        // R"(.*smoke_BehaviorTests.*OVExecGraphImportExportTest.*importExportedIENetwork.*type=(i64|u64).*)",
+
+	// Issue 66685
+        R"(smoke_PrePostProcess.*resize_linear_nhwc.*)",
```

Comment:
I suppose we can remove it since it's fixed?

Reply:
removed